src/CattyTests/Firebase/FirebaseCrashlyticsReporterTests.swift

```diff
@@ -21,125 +21,98 @@
  */
 
 import Firebase
+
 @testable import Pocket_Code
+
 import XCTest
 
-final class FirebaseCrashlyticsSetupTests: XCTestCase {
+final class FirebaseCrashlyticsReporterTests: XCTestCase {
 
-    var app: AppDelegate?
     var crashlytics: CrashlyticsMock?
+    var reporter: FirebaseCrashlyticsReporter?
 
     override func setUp() {
         super.setUp()
 
         crashlytics = CrashlyticsMock(collectionEnabled: false)
-        app = AppDelegateMock(crashlytics: crashlytics!)
+        reporter = FirebaseCrashlyticsReporter(crashlytics: crashlytics!)
+        UserDefaults.standard.set(true, forKey: kFirebaseSendCrashReports)
```

Comment:
Please change the order of those statements, otherwise the setup of the `FirebaseCrashlyticsReporter` might not work as excepted since it depends on the `UserDefaults` you are configuring.